docs/reference/sql/functions/date-time.asciidoc

```diff
@@ -487,6 +484,16 @@ include-tagged::{sql-specs}/docs/docs.csv-spec[dateTimeParse1]
 include-tagged::{sql-specs}/docs/docs.csv-spec[dateTimeParse2]
 --------------------------------------------------
 
+[NOTE]
+====
+If timezone is not specified in the datetime string expression and the parsing pattern, the resulting `datetime` will have the
+time zone specified by the user through the `time_zone`/`timezone` REST/driver parameters with no conversion applied.
```

Comment:
As an improvement, I would add links to REST and jdbc driver params documentation where these parameters are defined.